modules/exploits/multi/http/phpmailer_arg_injection.rb

```diff
@@ -68,9 +68,10 @@ def trigger(trigger_uri)
     while wait_time > 0
       sleep(sleep_time)
       wait_time -= sleep_time
-      res = send_request_cgi(
+      res = send_request_cgi!({
         'method'   => 'GET',
         'uri'      => trigger_uri
+        }, redirect_depth = 1
```

Comment:
The second arg is `timeout`, not `redirect_depth`, and the default is already 1.